modules/nw-enabling-hsts-per-route.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+// * networking/configuring-routing.adoc
+
+[id="nw-enabling-hsts-per-route_{context}"]
+= Enforcing HTTP strict transport security per-route
+
+HTTP strict transport security (HSTS) is implemented in the HAProxy template and applied to edge and reencrypt routes that have the `haproxy.router.openshift.io/hsts_header` annotation.
+
+If you do not have many routes or regulatory compliance requirements, you can use the per-route method of configuring HSTS.
```

Comment:
```suggestion
```

This is not accurate.  They have to use the per-route method no matter what.

Reply:
Thank you! I missed that one--will remove.